core/storage/user/gae_models.py

```diff
@@ -913,8 +913,8 @@ class UserSubscriptionsModel(base_models.BaseModel):
     """
 
     # IDs of activities (e.g., explorations) that this user subscribes to.
-    # TODO(#10727): Rename this to exploration_ids and perform a migration.
-    activity_ids = (
+    # IDs of explorations that this user subscribes to.
```

Comment:
Shouldn't you delete the comment that is one line above this one?

Reply:
Done.